jobs/uaa/spec

```diff
@@ -744,6 +744,16 @@ properties:
     description: "how long the results of a preflight request is cached"
     default: ~
 
+  # Content Security Policy (csp) settings
+  uaa.csp.script-src:
+    description: "Overrides the default script-src CSP header value of 'self'. Set this to allow scripts to be loaded from sources besides UAA. Because this overrides (not appends to) the value, be sure to include an entry for 'self' so scripts from UAA are allowed."
+    default:
+      - "'self'"
+    example:
+      - "'self'"
+      - "'unsafe-inline'"
+      - "js.example.com"
```

Comment:
I am confused that there are no single quotes around this URL.

Is it really that URLs don't need the single quotes, but the static strings `self` and `unsafe-inline` really require them?

Reply:
The last one does not need it, yes, so we can fix this